src/org-manager/org-manager.routing.ts

```diff
@@ -13,16 +13,24 @@ import { UserDetailsComponent } from './containers/user-details/user-details.com
 import { UserApprovalGuard } from './guards/users-approval.guard';
 import { DeleteOrganisationComponent } from './containers/delete-organisation/delete-organisation.component';
 import {DeleteOrganisationSuccessComponent} from './containers/delete-organisation-success/delete-organisation-success.component';
+import { RoleGuard } from '@hmcts/rpx-xui-common-lib';
 
 export const ROUTES: Routes = [
   {
-    path: 'active-organisation',
-    component: ActiveOrganisationsComponent,
-    canActivate: [AuthGuard],
+    path: 'organisation',
+    component: PendingOrganisationsComponent,
+    canActivate: [AuthGuard, RoleGuard],
+    data: {needsRole: ['prd-admin'], roleMatching: 0 }
```

Comment:
Same thing about the `roleMatching` property.

Reply:
Thats the change I made. When you use enum in Guard the PROD build fails

Reply:
This is fixed now.